cdap-spark-core/src/test/java/co/cask/cdap/app/runtime/spark/SparkTransactionServiceTest.java

```diff
@@ -173,7 +176,8 @@ public Transaction startLong() {
     };
     txManager.startAndWait();
     try {
-      SparkTransactionService sparkTxService = new SparkTransactionService(new InMemoryTxSystemClient(txManager));
+      SparkTransactionService sparkTxService = new SparkTransactionService(
```

Comment:
no way to simulate that the hostname is unavailable in unit tests right? because it uses ``InMemoryProgramRuntimeService``, and ``@Named(Constants.AppFabric.SERVER_ADDRESS)`` will always be set  by guice via ``cdap-default.xml``?